test/t.jl

```diff
@@ -92,7 +92,7 @@ import LinearAlgebra
         end
     end
     @testset "tkvrsn" begin
-        @test tkvrsn() == "CSPICE_N0066"
+        @test tkvrsn() == "CSPICE_N0067"
```

Comment:
I don't think checking an exact version number is very useful, especially when you allow multiple versions of the library.

Reply:
Do you think I should remove the test in the pull request? I just didn't want a test to be failing

Reply:
I'd at least do
```suggestion
        @test contains(tkvrsn(), r"CSPICE_N006(6|7}")
```
and keep v66 in the compat in Project.toml. Removing that broke older versions of Julia